tests/integration/targets/lambda/defaults/main.yml

```diff
@@ -4,3 +4,4 @@
 # we hash the resource_prefix to get a shorter, unique string
 lambda_function_name: '{{ tiny_prefix }}'
 lambda_role_name: 'ansible-test-{{ tiny_prefix }}-lambda'
+tiny_prefix: "d7er9e"
```

Comment:
tiny_prefix is a special variable provided by ansible-test and does not need to be defined

Reply:
opps! I didn't think that made it in there! My bad